<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>试验故障异常检测系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errordetection.qrc">
    <normaloff>:/images/images/timg.jpeg</normaloff>:/images/images/timg.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>部件</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>总报警面板</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>冷却水机组</string>
           </property>
           <item>
            <property name="text">
             <string>冷却水出水温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷却水出口水压</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷却水水流量</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷却水回水温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷却水机组水控制设定温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>冷却水压缩机排气温度</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>低温泵</string>
           </property>
           <item>
            <property name="text">
             <string>低温泵1级冷头温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵2级冷头温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵1#胀板温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵2#胀板温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵3#胀版温度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵供电状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵1级加热器运行状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵2级加热器运行状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温泵运行电流</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>分子泵</string>
           </property>
           <item>
            <property name="text">
             <string>分子泵电流</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>分子泵频率</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>分子泵状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>分子泵温度</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>热沉温度</string>
           </property>
           <item>
            <property name="text">
             <string>热沉平均温度</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>液氮泵</string>
           </property>
           <item>
            <property name="text">
             <string>低温系统1#液氮泵电流</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温系统1#液氮泵电压</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>低温系统1#液氮泵功率</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>液氮储槽</string>
           </property>
           <item>
            <property name="text">
             <string>储槽液位</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>储槽压力</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,2" columnstretch="1,3,1">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="4">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>214</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>  起始日期</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="4">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>214</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="3" column="3">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>截止日期</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0" colspan="4">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>62</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
           <item row="6" column="3">
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="0" colspan="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>198</width>
               <height>61</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="8" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>确定导出</string>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>62</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="0" colspan="4">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>198</width>
               <height>62</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QWidget" name="widget_3" native="true"/>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_2" native="true">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>报警面板</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>73</x>
             <y>80</y>
             <width>51</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>信号灯</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>270</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>取消报警</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>54</x>
             <y>330</y>
             <width>91</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>查看历史信息</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableWidget" name="tableWidget">
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <column>
           <property name="text">
            <string>年月</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>时刻</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>部件</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数值</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>评价</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_6"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_6"/>
   <addaction name="action_5"/>
   <addaction name="separator"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="errordetection.qrc">
     <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
   </property>
   <property name="text">
    <string>设置阈值</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="errordetection.qrc">
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="errordetection.qrc">
     <normaloff>:/images/images/find.png</normaloff>:/images/images/find.png</iconset>
   </property>
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="errordetection.qrc">
     <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>设置读取时间</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>总报警面板</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>更改为离线模式</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>更改为实时模式</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="errordetection.qrc"/>
 </resources>
 <connections/>
</ui>
